examples/unit-test/mbed-blink/platformio.ini

```diff
@@ -0,0 +1,15 @@
+# ; PlatformIO Project Configuration File
+# ;
+# ;   Build options: build flags, source filter, extra scripting
+# ;   Upload options: custom port, speed and extra flags
+# ;   Library options: dependencies, extra library storages
+# ;
+# ; Please visit documentation for the other options and examples
+# ; http://docs.platformio.org/en/stable/projectconf.html
+
+[env:bluepill_f103c8]
+platform = ststm32
+board = bluepill_f103c8
+framework = mbed
+upload_protocol = gdb
+test_port = /dev/cu.usbmodemBFDEA5F3
```

Comment:
What to do with this port? I'm not sure that each bluepill_f103c8 board has "2 Serial Ports"